xds/internal/xdslbregistry/converter_test.go

```diff
@@ -349,7 +345,7 @@ func (s) TestConvertToServiceConfigFailure(t *testing.T) {
 		},
 		// TODO: test validity right on the boundary of recursion 16 layers
 		// total.
```

Comment:
Can you not just delete one `wrrLocality` from below and take care of this in 30 seconds?

Reply:
One below is like 20+, but I can add, but it seems like a useless test case.

Reply:
So delete 5 or whatever.

If it's useless, why have a TODO?

Reply:
Also, if it's important to test the boundary condition (to have the same behavior as c and java), then we should test [allowable_limit, allowable_limit+1] not [..., allowable_limit+5].

Reply:
K added test and switched this test to + 1.